R/standardize.models.R

```diff
@@ -273,7 +273,13 @@ standardize.default <- function(
   on.exit(.update_failed())
 
   if (isTRUE(verbose)) {
-    model_std <- eval(substitute(update_expr))
+    # Remove this suppressWarnings() and therefore this ifelse when
+    # https://github.com/lrberge/fixest/issues/618 is fixed.
+    if (inherits(x, "fixest")) {
```

Comment:
Why is this needed only for fixest?
Warnings probably should be displayed - they might be informative to the user.